mslib/msui/_tests/test_mscolab.py

```diff
@@ -211,16 +247,44 @@ def test_set_exported_file(self, mockopen, mockmessage):
         QtWidgets.QApplication.processEvents()
         assert self.window.listProjects.model().rowCount() == 1
 
-    def test_add_project(self):
+    @mock.patch("PyQt5.QtWidgets.QErrorMessage")
+    def test_add_project(self, mockbox):
         # ToDo test needs to be independent from test_user_delete
```

Comment:
If this is now independent, so that each can run without the other the comment can be removed